src/pages/Register/styles.jsx

```diff
@@ -0,0 +1,90 @@
+import styled from "styled-components";
+import { mobile } from "../../responsive";
```

Comment:
i didnt knew you could do this

Reply:
yeah, I want to create one project with a styled component so I tried this one.